### PR TITLE
feat: Alert Component

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Please review this PR.
Alert component with different message.

link: https://optimistic-engelbart-3c75b6.netlify.app/

### CODE REVIEW

**Comment by Namrata-J** [comment]
it looks good, you can merge it!

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: css/style.css [hunk 1]
```diff
@@ -71,11 +73,18 @@
...
 .sidebar a {
-    border-bottom: 2px solid #f3e6b0;
-    padding-bottom: 0.5rem;
+  border-bottom: 2px solid #f3e6b0;
+  padding-bottom: 0.5rem;
+  font-size: 1.1rem;
+}
+
+.hd-3 {
+  margin: 1rem 0 0.2rem 0;
```
Comment (fahd23) [inline css/style.css:82]: if margin property has 3 values, you can add like this:- margin: 1rem 0 0.2rem;

**Review by fahd23 — APPROVED** [review]
Looks good, you can merge it

File: css/style.css (hunk 1 above)
Comment (ozairkhan2810) [reply css/style.css:82]: Ok! noted.
thank you for this

**Pull request merged**